reaper/reaper.go

```diff
@@ -66,12 +69,26 @@ func NewReaper() *Reaper {
 	}
 }
 
+func GetPrices() {
+	// prevent shadowing
+	var err error
+	log.Info("Downloading prices")
+	pricesMap, err = prices.DownloadPricesMap(prices.Ec2PricingUrl)
+	if err != nil {
+		log.Error(fmt.Sprintf("Error getting prices: %s", err.Error()))
+	}
+	log.Info("Successfully downloaded prices")
+}
+
 // Start begins Reaper's schedule
 func (r *Reaper) Start() {
 	// adding as a job runs r.Run() every interval
 	r.Cron.Schedule(cron.Every(config.Notifications.Interval.Duration), r)
 	r.Cron.Start()
 
+	// start prices download...
+	go GetPrices()
```

Comment:
Does anything depend on the price data being available? If it does, this should be run synchronously so it doesn't cause a race condition.